starboard/nplb/media_configuration_test.cc

```diff
@@ -25,9 +27,15 @@ TEST(SbMediaConfigurationTest, ValidatePerformance) {
 
   const int count_audio_output = SbMediaGetAudioOutputCount();
   for (int i = 0; i < count_audio_output; ++i) {
+    constexpr int kNumberOfCalls = 100;
+    // TODO(b/284140486): Optimize SbMediaGetAudioConfiguration() to reduce the
```

Comment:
Please make a new bug for this, as this bug seems it will be closed before this is potentially optimized.

Reply:
Please see b/284140486#comment19, we'll investigate the optimization before close it.